ti/instrumentation/python/unittest.go

```diff
@@ -52,7 +53,7 @@ func (m *unittestRunner) ReadPackages(workspace string, files []ti.File) []ti.Fi
 func (m *unittestRunner) GetCmd(ctx context.Context, tests []ti.RunnableTest, userArgs, workspace,
 	agentConfigPath, agentInstallDir string, ignoreInstr, runAll bool) (string, error) {
 	if userArgs == "" {
-		userArgs = "--junitxml=harness_test_results.xml -o junit_family='xunit1'"
+		userArgs = fmt.Sprintf("--junitxml='%s${HARNESS_NODE_INDEX}' -o junit_family='xunit1'", common.HarnessDefaultReportPath)
```

Comment:
You have the value here. Just use it instead of relying on the script.
```suggestion
		userArgs = fmt.Sprintf("--junitxml='%s' -o junit_family='xunit1'", common.HarnessDefaultReportPath, getHarnessDefaultXml())
```